cdap-docs/admin-manual/source/installation/cloudera/installation.rst

```diff
@@ -61,7 +61,7 @@ Run the Cloudera Manager Admin Console *Add Service* Wizard and select *CDAP*.
 When completing the Wizard, these notes may help:
 
 - *Add Service* Wizard, Page 2: **Optional Hive dependency** is for the optional CDAP
```

Comment:
It's true that Hive is optional, but by default it's needed. This is confusing since we don't have anything explicitly telling the admin to add the Hive dependency unless they disable explore.

We should include the Hive dependency by default and allow users to disable explore/Hive. Then, we can add documentation about this.